Update the documentation to install the hooks (#26485)
DataDog/datadog-agent: docs/dev/agent_dev_env.md

![deva_install](../public/assets/images/deva.gif)

> We also provide an invoke task to perform some sanity checks and enable some tools. To use it you can run:

Reviewer: ```suggestion
Datadog also provides an invoke task to perform some sanity checks and enable some tools. To use it, run:
```

#### Installation

> To install pre-commit, follow [these instructions](https://pre-commit.com/#installation). The `deva setup` will automatically enable the hooks but you can do it manually running: `pre-commit install`.

Reviewer: ```suggestion
To install pre-commit, follow [these instructions](https://pre-commit.com/#installation). The `deva setup` automatically enables the hooks, but you can do it manually running: `pre-commit install`.
```
